modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/challenge/questions/ServerChallengeTestCase.java

```diff
@@ -0,0 +1,346 @@
+/*
+ *Copyright (c) 2005-2010, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *WSO2 Inc. licenses this file to you under the Apache License,
+ *Version 2.0 (the "License"); you may not use this file except
+ *in compliance with the License.
+ *You may obtain a copy of the License at
+ *
+ *http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *Unless required by applicable law or agreed to in writing,
+ *software distributed under the License is distributed on an
+ *"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *KIND, either express or implied.  See the License for the
+ *specific language governing permissions and limitations
+ *under the License.
+ */
+package org.wso2.identity.integration.test.rest.api.challenge.questions;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.testng.Assert;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import org.wso2.identity.integration.test.rest.api.challenge.questions.model.ServerChallengeModel;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static io.restassured.RestAssured.given;
+
+/**
+ * This class is for ServerChallengeQuestion rest api test cases.
+ */
+public class ServerChallengeTestCase {
+
+
+    private final String username = "admin";
```

Comment:
Use all caps with an underscore between words for constants.

Reply:
done